bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -117,6 +118,7 @@ public Response doCreate( final String packageType, final String type, final Str
                 builderModifier.accept( builder );
             }
             response = builder.build();
+            contentController.generateHttpMetadataHeaders( transfer, request, response );
         }
```

Comment:
👍 

Can we move the HttpExchangeMetadata construction logic that uses Response and HttpServletRequest into an Indy utility class somewhere, maybe in indy-bindings-jaxrs, so we don't have to have additional dependencies in the galley-transport-htcli module...which could still work in a system that doesn't involve either JAX-RS or the Servlets API?